JT/01_environment_setup/README.adoc

```diff
@@ -0,0 +1,406 @@
+
+image::resources/idea-03.png[]
+
+
+. Install plugins for IntelliJ IDEA
++
+* Gherkin
+* Cucumber for Java
++
+image::resources/idea-04.png[]
++
+* Lombok
++
+image::resources/idea-06.png[]
+
+. Set up the new project in IntelliJ IDEA or create a new one from existing sources
++
+I. You should have a link to the project repository. If you don't have it, ask your team lead for it.
++
+II. OR:
+* Set a project name.
+* Choose where the project will be created.
+* Choose Language: Java.
+* Choose Build system: Maven.
+* Choose SDK: 21 (or the version you have installed).
+* Click Finish.
+
+image::resources/idea-07.png[]
+
+* Open the project structure and set the project SDK and language level (Press Ctrl+Alt+Shift+S).
++
+image::resources/idea-08.png[]
++
+image::resources/idea-09.png[]
+
+=== [[postman]]Postman
+
+Postman is an app for interacting with HTTP APIs. It presents you with a friendly GUI for constructing requests and reading responses.
+
+But Postman recently changed their software to remove the Scratch Pad mode, which means that all configuration data, including Collections and Environments, is uploaded to Postman's cloud servers. This poses a **potential security risk** for many projects. To avoid storing collections in a Postman account, users are advised to uninstall their current version of Postman and install an older version using the following link:
+
+link:https://go.pstmn.io/dl-win64-v9-latest[https://go.pstmn.io/dl-win64-v9-latest]
+
+. Download Postman for Windows (64-bit), from the following link: https://go.pstmn.io/dl-win64-v9-latest.
++
+image::resources/postman-01.png[]
+
+. Install Postman
++
+image::resources/postman-02.png[]
+
+**Do not sign up or log in and don't Update software for security reasons.**
+
+Requests are the foundation of Postman, as they are used to test APIs. Over time, the number of requests will increase, and it can become challenging to navigate through them. Therefore, it's essential to organize requests into collections. Collections can be thought of as folders for storing requests. Organizing collections is a significant topic, and it's something that might be discussed in an interview. In short, collections should have a logical structure and be organized based on a certain criterion, such as the version of the API being tested or a specific block of functionality.
+
+Both requests and collections can be created using the "New" button in the upper left corner of the interface.
+
+Postman collection with endpoints and environments set up will be provided by your project team lead.
+
+=== [[winscp]]WinSCP
+
+WinSCP (Windows Secure Copy) is a free and open-source file transfer client for Windows. It supports protocols like SFTP, SCP, FTPS, and FTP, and is commonly used for secure file transfer between a local computer and a remote server. It also offers a graphical user interface for easy navigation and file management.
+
+. Download WinSCP
```

Comment:
Could add some screenshots for the WinSCP? U added screens for other tools above and it would be nice to continue this tradition.

Reply:
Fair enough. I have added 4 pictures 4 tradition